src/Collection/CollectionBuilder.php

```diff
@@ -71,6 +71,21 @@ public function __construct($commandFile)
     }
 
     /**
+     * @param int $logLevel
+     *
+     * @return $this
+     */
+    public function setLogLevel($logLevel)
```

Comment:
The collection builder will already call through to any method of the active task via the `_call()` special function. However, I have to re-consider my recommendation to account for multiple task support.